contracts/MerkleLib.sol

```diff
@@ -46,6 +46,8 @@ library MerkleLib {
         bytes32[] memory proofs
     ) internal pure returns (bool) {
         bytes32 hash = dataHash;
+        if (proofs.length != nChunkBits) return false;
```

Comment:
Looks like we could remove `nChunkBits` argument and replace it with `proofs.length`?

Reply:
Yes, less calldata